src/components/Comments/WriteComment.js

```diff
@@ -78,9 +78,15 @@ const WriteComment = ({
       <View style={styles.rowTop}>
         <View style={styles.replyToContainer}>
           <Text style={styles.replyToTitle}>Reply to </Text>
-          <Text style={styles.replyToUsername}>{username}</Text>
-          {value?.trim() !== '' && (
-            <TouchableOpacity onPress={() => onChangeText('')}>
+          <Text style={styles.replyToUsername} numberOfLines={1} ellipsizeMode="tail">
+            {username}
+          </Text>
+          {(isReply || value?.trim() !== '') && (
+            <TouchableOpacity
+              onPress={() => {
+                onChangeText('');
+                onClear();
+              }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for displaying the close button has been changed to include `isReply`. This seems fine as long as the `isReply` prop is being correctly set in all cases where this component is used. Also, the `onClear` function is now being called when the close button is pressed. As mentioned above, ensure that this function is always provided by the parent component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->